src/pre_process/m_initial_condition.fpp

```diff
@@ -181,7 +181,7 @@ contains
 
                     ! 3D STL patch
                 elseif (patch_icpp(i)%geometry == 21) then
-                    call s_model(i, patch_id_fp, q_prim_vf)
+                    call s_model(i, patch_id_fp, q_prim_vf, .false.)
```

Comment:
i don't think we want to be calling functions with a 'magic' value like `.false.`. instead, you should define or use an existing boolean for this argument.